build/dependencies.props

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <!-- Partner Versions-->
   <PropertyGroup>
-    <CodeAnalysisVersion>2.3.0-beta1</CodeAnalysisVersion>
+    <CodeAnalysisVersion>2.3.0-beta3-61731-06</CodeAnalysisVersion>
```

Comment:
This version does not ship on NuGet. Us in Razor use this version for our VisualStudio only components because that's what's shipped in VS.

Lets talk more with @rynowak I know he had concerns about Scaffolding's use of this package 😄